source/system/Buffer.ooc

```diff
@@ -49,7 +49,7 @@ Buffer: cover {
 			start = this _size + start
 		if (distance < 0)
 			start -= size
-		start < this size ? This new(this _pointer + start, size minimum(this _size - start)) : This empty
+		start < this size ? This new(this _pointer as Byte* + start, size minimum(this _size - start)) : This empty
```

Comment:
That's right, because the expression `*(p + x)`, where `p` is a pointer `T*` and `x` integer, generates memory access at address `p + x * sizeof(T)`, and for `T = void` `sizeof(void)` is not defined.